nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/AuthenticationConfigurationDTO.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.web.api.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.xml.bind.annotation.XmlType;
+
+/**
+ * Authentication Configuration endpoint and status information
+ */
+@XmlType(name = "authenticationConfiguration")
+public class AuthenticationConfigurationDTO {
+
+    private boolean loginSupported;
+
+    private String loginUri;
+
+    private String logoutUri;
```

Comment:
It would be helpful to additional know when the `loginUri` is our own local hosted UI or an external log in (like SSO). This would allow the front end to more gracefully handle the routing.

Reply:
Thanks for the feedback @mcgilman, I added an `externalLoginRequired` field indicating the status of the configured login URI.